packages/ng-table-builder/src/table-builder.component.html

```diff
@@ -22,13 +22,13 @@
     (recalculatedHeight)="recalculateHeight()"
     [autoHeight]="{
         detect: autoHeightDetect,
-        height: height,
+        height: rootHeight,
```

Comment:
can we rename then `height` to `rootHeight`?

Reply:
renamed